nncf/quantization/algorithms/hyperparameter_tuner/pipelines.py

```diff
@@ -0,0 +1,132 @@
+
+from nncf.common.quantization.structs import QuantizationPreset
+from nncf.data.dataset import Dataset
+from nncf.parameters import ModelType
+from nncf.parameters import TargetDevice
+from nncf.quantization.advanced_parameters import AdvancedQuantizationParameters
+from nncf.quantization.algorithms.accuracy_control.evaluator import MetricResults
+from nncf.quantization.algorithms.hyperparameter_tuner.algorithm import HyperparameterTuner
+from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantization
+from nncf.quantization.range_estimator import AggregatorType
+from nncf.quantization.range_estimator import RangeEstimatorParameters
+from nncf.quantization.range_estimator import StatisticsCollectorParameters
+from nncf.quantization.range_estimator import StatisticsType
+from nncf.scopes import IgnoredScope
+
+TModel = TypeVar("TModel")
+TTensor = TypeVar("TTensor")
+
+
+def get_quantization_param_grid() -> Dict[str, Any]:
+    """
+    Returns params grid for post-training quantization algorithm.
+    """
+    min_param_values = [
+        StatisticsCollectorParameters(
+            statistics_type=StatisticsType.MIN,
+            aggregator_type=AggregatorType.MIN,
+        ),
+        StatisticsCollectorParameters(
+            statistics_type=StatisticsType.QUANTILE,
+            aggregator_type=AggregatorType.MEAN,
+            quantile_outlier_prob=10e-4,
+        ),
+        StatisticsCollectorParameters(
+            statistics_type=StatisticsType.QUANTILE,
+            aggregator_type=AggregatorType.MEAN,
+            quantile_outlier_prob=10e-5,
+        ),
+    ]
+    max_param_values = [
+        StatisticsCollectorParameters(
+            statistics_type=StatisticsType.MAX,
+            aggregator_type=AggregatorType.MAX,
+        ),
+        StatisticsCollectorParameters(
+            statistics_type=StatisticsType.QUANTILE,
+            aggregator_type=AggregatorType.MEAN,
+            quantile_outlier_prob=10e-4,
+        ),
+        StatisticsCollectorParameters(
+            statistics_type=StatisticsType.QUANTILE,
+            aggregator_type=AggregatorType.MEAN,
+            quantile_outlier_prob=10e-5,
+        ),
+    ]
+
+    param_grid = {
+        "preset": [QuantizationPreset.PERFORMANCE, QuantizationPreset.MIXED],
+        "fast_bias_correction": [True, False],
+        "advanced_parameters:weights_range_estimator_params": [
+            RangeEstimatorParameters(
+                min=StatisticsCollectorParameters(statistics_type=StatisticsType.MIN),
+                max=StatisticsCollectorParameters(statistics_type=StatisticsType.MAX),
+            )
+        ],
+        "advanced_parameters:activations_range_estimator_params": [
+            RangeEstimatorParameters(min=min_v, max=max_v)
+            for min_v, max_v in itertools.product(min_param_values, max_param_values)
+        ],
+    }
+
+    return param_grid
+
+
+def quantize_with_tune_hyperparams(
```

Comment:
I would suggest to move `quantize_with_tune_hyperparams` to the `openvino/quantization/quantize_model.py`.

Reply:
Done